.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+Thanks for contributing!
+-->
+
+Some things to make sure you have completed:
+- [ ] Bump Spec version.
+- [ ] Update last modified date.
+- [ ] Update changelog.
+- [ ] Make sure there is generated JSON files from the YAML test files.
+- [ ] Test changes in your language driver.
```

Comment:
The person writing the spec is not always the POC implementer, and we often request POCs in multiple languages.

```suggestion
- [ ] Test changes in at least one language driver.
```